repo/packages/K/kubernetes/1/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "kubernetes",
+  "version": "v1.0.5-v0.6.4-alpha",
+  "scm": "https://github.com/mesosphere/kubernetes-mesos",
+  "maintainer": "support@mesosphere.io",
+  "description": "Manage a cluster of Linux containers as a single system to accelerate Dev and simplify Ops.",
+  "framework": true,
+  "images": {
+    "icon-small": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-48x48.png",
+    "icon-medium": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-96x96.png",
+    "icon-large": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-256x256.png"
+  },
+  "tags": ["mesosphere", "framework", "pod", "label", "paas"],
```

Comment:
i remember suggesting `kubernetes` but was told that since that's the name
of the package it would be redundant here. `google` seems like a nice
addition

On Thu, Sep 17, 2015 at 6:02 PM, Ken Sipe <notifications@github.com> wrote:

> In repo/packages/K/kubernetes/1/package.json
> <https://github.com/mesosphere/universe/pull/200#discussion_r39804674>:
>
> > @@ -0,0 +1,23 @@
> > +{
> > +  "name": "kubernetes",
> > +  "version": "v1.0.5-v0.6.4-alpha",
> > +  "scm": "https://github.com/mesosphere/kubernetes-mesos",
> > +  "maintainer": "support@mesosphere.io",
> > +  "description": "Manage a cluster of Linux containers as a single system to accelerate Dev and simplify Ops.",
> > +  "framework": true,
> > +  "images": {
> > +    "icon-small": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-48x48.png",
> > +    "icon-medium": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-96x96.png",
> > +    "icon-large": "https://downloads.mesosphere.io/kubernetes/artifacts/images/k8s-256x256.png"
> > +  },
> > +  "tags": ["mesosphere", "framework", "pod", "label", "paas"],
>
> shouldn't this contain kubernetes? perhaps google?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesosphere/universe/pull/200/files#r39804674>.
>